### PR TITLE
JENKINS-51631: Rebranding some remaining expression + adding version …

### PR DESCRIPTION
**Pull request by danieldanan**
…of plugin in logs (already implemented)

### CODE REVIEW

File: src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java [hunk 1]
```diff
@@ -224,9 +225,27 @@ public static String getPcreportFileName() {
...
+    private String getVersion() {
+        try {
+            String completeVersion = ConfigurationService.getPluginVersion();
+            if(completeVersion != null) {
+                String[] partsOfCompleteVersion = completeVersion.split(" [(]");
+                if (partsOfCompleteVersion != null)
+                    return partsOfCompleteVersion[0];
+            }
+        }
+        catch(Exception e)
```
Comment (grount) [inline src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java]: Please use only the exception that is being thrown and not swallowing all the unreleased exceptions, split throws PatternSyntaxException and please log the issues as well.

File: src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java [hunk 2]
```diff
@@ -224,9 +225,27 @@ public static String getPcreportFileName() {
         return pcReportFileName;
     }
 
+    private String getVersion() {
+        try {
+            String completeVersion = ConfigurationService.getPluginVersion();
+            if(completeVersion != null) {
+                String[] partsOfCompleteVersion = completeVersion.split(" [(]");
+                if (partsOfCompleteVersion != null)
```
Comment (grount) [inline src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java]: I am not sure can split return null?

File: src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java [hunk 3]
```diff
@@ -224,9 +225,27 @@ public static String getPcreportFileName() {
...
+        {
+        }
+        return "unknown";
+    }
+
     private Testsuites execute(PcClient pcClient, Run<?, ?> build)
             throws InterruptedException,NullPointerException {
         try {
+            String pomVersionget = getVersion();
+            if(!(pomVersionget == null || pomVersionget =="unknown"))
```
Comment (grount) [inline src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java]: Have you seen codacy suggestion?

**Comment by danieldanan** [comment]
thanks for the feedback. modifying code.

**Comment by grount** [comment]
Approved for test build

**Pull request merged**